src/App.js

```diff
@@ -1,15 +1,40 @@
-import React from 'react';
+import { React, useState } from 'react';
 import './App.css';
+import ChatEntry from './components/ChatEntry';
 import chatMessages from './data/messages.json';
-
+import ChatLog from './components/ChatLog'
 const App = () => {
+
+  const [messageData, setMessageData] = useState(chatMessages)
+  const [likeData, setLikeData] = useState(0)
+
+  const updateMessage = (messageToUpdate) => {
+    const entries = messageData.map((message) => {
+      if (message.id === messageToUpdate.id) {
+        return messageToUpdate
+      }
```

Comment:
hmm I'm a little confused by this if statement and variable. I don't see you using `entries` anywhere. Do you need this? It's returning the message that corresponds with the message we want to update, but we already have it.

You are returning the same thing  on line 14 that you already brought in in the param on line 11

I think all you need is lines 16-22